packages/metadata/src/providers/dummy.provider.ts

```
@@ -0,0 +1,10 @@
+import type { IMetadataProvider } from "../internal.js";
+
+export class DummyMetadataProvider implements IMetadataProvider {
+    constructor() {}
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)